core/src/main/java/io/aiven/klaw/service/TopicControllerService.java

```diff
@@ -153,8 +156,17 @@ private ApiResponse createTopicRequest(TopicRequestModel topicRequestReq) throws
     mapAdvancedTopicConfiguration(topicRequestReq, topicRequestDao);
     topicRequestDao.setTenantId(commonUtilsService.getTenantId(userName));
 
-    String result = dbHandle.requestForTopic(topicRequestDao).get("result");
-    // default to Topic_CREATE which is the old hard coded
+    String result;
+    try {
+      result = dbHandle.requestForTopic(topicRequestDao).get("result");
```

Comment:
Is it possible for you to add a validation on the front end to restrict to length of db column of description ?

Reply:
I added the constraint to the coral Zod validation schema in Coral. The old version already had it.